enterprise/cli/proxyserver.go

```diff
@@ -114,6 +114,8 @@ func (r *RootCmd) proxyServer() *serpent.Command {
 		),
 		Handler: func(inv *serpent.Invocation) error {
 			var closers closers
+			defer closers.Close()
```

Comment:
Lol, `closers` was a nice though, but never closing the closers is quite the funny bug.

Reply:
```go
var losers closers
defer losers.Close()
```

:)